packages/template-retail-react-app/app/components/product-view-modal/bundle.jsx

```diff
@@ -32,21 +32,30 @@ import {useDerivedProduct} from '@salesforce/retail-react-app/app/hooks'
 const BundleProductViewModal = ({product: bundle, isOpen, onClose, updateCart, ...props}) => {
     const productViewModalData = useProductViewModal(bundle)
     const {variationParams} = useDerivedProduct(bundle)
+    const childProductRefs = useRef({})
+    const [childProductOrderability, setChildProductOrderability] = useState({})
+    const [selectedChildProducts, setSelectedChildProducts] = useState([])
+    const [selectedBundleQuantity, setSelectedBundleQuantity] = useState(
+        productViewModalData?.product?.quantity
+    )
+    const trueIfMobile = useBreakpointValue({base: true, lg: false})
 
-    const childProductIds = productViewModalData.product?.bundledProductItems
+    let childProductIds = productViewModalData.product?.bundledProductItems
         ?.map(({productId}) => productId)
         .join(',')
+    let productIds = selectedChildProducts.map(({variant}) => variant.productId).join(',')
```

Comment:
`childProductIds` is re-declared (so `let` is correct) but `productIds` appears not to be re-declared, so `let` would be correct on `43` and wrong on `46`, I believe